app/dashboard/templates/profiles/header_details.html

```diff
@@ -64,26 +64,32 @@ <h1 class="profile-header__handle">
   <button class="btn btn-gc-blue btn-sm" data-jointribe="{{profile.handle}}">Join Tribe</button>
 {% endif %}
 
-
-{% if 0 %}
-TODO: Re-enable this when we have proper privacy in place a la https://github.com/gitcoinco/web/issues/5654
+{% if not profile.hide_wallet_address %}
```

Comment:
maybe instead add to the next condition "and not profile.hide_wallet_address"

Reply:
yes this..

Reply:
@iamonuwa why was this marked as resolved?

Reply:
This issue has been fixed by @thelostone-mc @danlipert